### ISSUE TITLE
[Question] Tuned, To be tuned, nothing

### ISSUE THREAD

**Issue by albertcthomas** [DLR-RM/rl-baselines3-zoo#340]
### ❓ Question

Hi,

I wanted to ask about the meaning of "Tuned", "To be tuned" and when none of these two are used in the hyperparams config files. For instance in `sac.yml`, for some envs we have "Tuned", for other "To be tuned" and for the rest nothing. #185 is kind of answering for "Tuned" I am not sure the difference between "To be tuned" and nothing is clear.

Thanks a lot!

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/rl-baselines3-zoo/issues) in the repo
- [X] I have read the [SB3 documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have read the [RL Zoo README](https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/README.md)
- [X] If code there is, it is minimal and working
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
Hello,

> https://github.com/DLR-RM/rl-baselines3-zoo/issues/185 is kind of answering for "Tuned" I am not sure the difference between "To be tuned" and nothing is clear.

you are right, the comments are not fully consistent.
As a rule of thumb, we can take the example of [sac](https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/hyperparams/sac.yml):
- "tuned" means that it was either tuned automatically with optuna or manually (usually a bit of both) and that the performance is satisfying (reaches good performance "fast enough")
- "not tuned" means that the current hyperparameters are not satisfying w.r.t. performance (compared to other algorithms)
- nothing either mean this it is using known good defaults (most of the time) or that I forgot to write "not tuned" (normally there should be very few of those). As an example, there is nothing written for `Pendulum-v1` because the SAC defaults work well for that env (I increased the learning rate to make it converge faster but this is not needed to reach good performance).

In case of doubt, best is to take a look at the [benchmark file](https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/benchmark.md) which tells you what score to expect if you use the hyperparameters of the RL Zoo (it also displayed on each huggingface hub page).

**Comment by albertcthomas** [comment]
Thanks a lot for the quick reply!

**Comment by albertcthomas** [comment]
> In case of doubt, best is to take a look at the [benchmark file](https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/benchmark.md) which tells you what score to expect if you use the hyperparameters of the RL Zoo (it also displayed on each huggingface hub page).

I was actually interested in the performance of SAC on Hopper (https://huggingface.co/sb3/sac-Hopper-v3) which seems worse than many other algorithms such as ARS, TD3 or TRPO. Therefore I was just curious how the hyperparameteres were obtained when none of "tuned" or "to be tuned" was written in the config file.

**Comment by albertcthomas** [comment]
PS: Thank you for this great resource/benchmark!

**Comment by araffin** [comment]
> which seems worse than many other algorithms such as ARS, TD3 or TRPO.

it does look similar, just a higher variance in the result (this is just one seed anyway).
For MuJoCo, we use hyperparameters from the paper (which are actually the default ones).

**Comment by albertcthomas** [comment]
> it does look similar, just a higher variance in the result (this is just one seed anyway).

Yes indeed, I'm obtaining comparable results.

**Comment by qgallouedec** [comment]
I take the occasion to advertise here, but if you are interested, feel free to share your results by contributing to the benchmark that we are building: https://github.com/openrlbenchmark/openrlbenchmark/issues/7

**Comment by albertcthomas** [comment]
> I take the occasion to advertise here, but if you are interested, feel free to share your results by contributing to the benchmark that we are building: [openrlbenchmark/openrlbenchmark#7](https://github.com/openrlbenchmark/openrlbenchmark/issues/7)

Good to know! I was not aware of this great initiative, I also saw the tweet by @araffin advertising it. What's the difference between the benchmark here (rl-baselines3-zoo) and the one at openrlbenchmark?

**Comment by qgallouedec** [comment]
Here, we just reported the final score of a single training. In openrlbenchamark, we report all the metrics across the entire training, and for multiple trainings.

**Comment by qgallouedec** [comment]
We don't have a clear policy on this yet. Here is what I propose:
- The first thing to do is to run as many experiments as in openrlbenchamrk so that we can be sure that the new set of hyperparameters is better (for now, I set it to 10 runs arbitrarily, but it seems reasonable). Do it in a separate wandb project.
- Then you can propose that these new hyperparameters become the default ones by opening a PR on this repo, and link your wandb project. 
- When merging, we import your results into openrlbenchmark, and we tag the previous results as "outdated".

**Comment by albertcthomas** [comment]
Thanks for the process. Not exactly sure when I'll be able to do this so in the meantime I'll report my findings in a new issue so that we have a trace.
